crengine/src/lvrend.cpp

```diff
@@ -3067,17 +3078,22 @@ void renderFinalBlock( ldomNode * enode, LFormattedText * txform, RenderRectAcce
                 while ( sibling ) {
                     lString32 marker;
                     int markerWidth = 0;
-                    if ( sibling->getNodeListMarker( counterValue, marker, markerWidth ) ) {
-                        if ( markerWidth > maxWidth )
-                            maxWidth = markerWidth;
+                    if ( sibling->getNodeListMarker( counterValue, marker ) ) {
+                        LVFontRef font = sibling->getFont();
+                        if ( !font.isNull() ) {
+                            TextLangCfg * lang_cfg = TextLangMan::getTextLangCfg(sibling);
+                            int markerWidth = font->getTextWidth(marker.c_str(), marker.length(), lang_cfg) + font->getSize()/8;
+                            if ( markerWidth > maxWidth )
+                                maxWidth = markerWidth;
```

Comment:
May be this (from a few lines above what I reference) could go thru listItemMarkerMaxWidth() ? as it seems we do the same kind of work.

Reply:
Yes, after looking over that gain, thanks.